package.json

```diff
@@ -61,11 +64,11 @@
     "include": [
       "server/**"
     ],
-    "check-coverage": false,
-    "branches": 50,
-    "lines": 50,
-    "functions": 50,
-    "statements": 50
+    "check-coverage": true,
+    "branches": 70,
+    "lines": 70,
+    "functions": 70,
+    "statements": 70
   },
   "proxy": "http://localhost:3001",
   "description": "This is a template for using Express and React in the same project. It is based on Create React App.",
```

Comment:
Not a big deal or a blocker at all but maybe we want to update this description lol

Reply:
I can do that in my next round of testing updates!